cmd/create/operatorroles/by_clusterkey.go

```diff
@@ -28,6 +28,17 @@ func handleOperatorRoleCreationByClusterKey(r *rosa.Runtime, env string,
 		os.Exit(1)
 	}
 
+	// Check to see if IAM operator roles have already created
+	missingRoles, err := validateOperatorRoles(r, cluster)
+	if err != nil {
+		if strings.Contains(err.Error(), "AccessDenied") {
```

Comment:
@hunterkepley I know you're just moving the code around, but any idea why this is like this? If the error contains `AccessDenied`, we just debug log it and move on, but otherwise we `os.Exit()`?

Reply:
@robpblake I think, in the case there are no permissions, we want to attempt to move on and run manual mode and return commands as the user may still want the commands. But, when auto mode, it will fail regardless later if no permissions.

In other cases, such as the normal 'roles exist' error, it will os.Exit @robpblake

Reply:
@hunterkepley imho, it's better to add this as the code comment